communication/obs-studio/basic/scenes/Untitled.json

```diff
@@ -0,0 +1,843 @@
+            "autoStartRecordTimer": false,
+            "autoStartStreamTimer": false,
+            "recordTimerHours": 0,
+            "recordTimerMinutes": 0,
+            "recordTimerSeconds": 30,
+            "streamTimerHours": 0,
+            "streamTimerMinutes": 0,
+            "streamTimerSeconds": 30
+        },
+        "scripts-tool": []
+    },
+    "name": "Untitled",
+    "preview_locked": false,
+    "quick_transitions": [
+        {
+            "duration": 300,
+            "hotkeys": [],
+            "id": 1,
+            "name": "Cut"
+        },
+        {
+            "duration": 300,
+            "hotkeys": [],
+            "id": 2,
+            "name": "Fade"
+        }
+    ],
+    "saved_projectors": [],
+    "scaling_enabled": false,
+    "scaling_level": 0,
+    "scaling_off_x": 0.0,
+    "scaling_off_y": 0.0,
+    "scene_order": [
+        {
+            "name": "MoC"
+        },
+        {
+            "name": "Office Hours"
+        },
+        {
+            "name": "office splash"
+        },
+        {
+            "name": "Community Meeting Splash"
+        },
+        {
+            "name": "Community Meeting"
+        }
+    ],
+    "sources": [
+        {
+            "balance": 0.5,
+            "deinterlace_field_order": 0,
+            "deinterlace_mode": 0,
+            "enabled": true,
+            "flags": 0,
+            "hotkeys": {},
+            "id": "text_ft2_source",
+            "mixers": 0,
+            "monitoring_type": 0,
+            "muted": false,
+            "name": "Community meeting",
+            "private_settings": {},
+            "push-to-mute": false,
+            "push-to-mute-delay": 0,
+            "push-to-talk": false,
+            "push-to-talk-delay": 0,
+            "settings": {
+                "color1": 4287056416,
+                "color2": 4287056416,
+                "custom_width": 2,
+                "drop_shadow": false,
+                "font": {
+                    "face": "Ubuntu",
+                    "flags": 0,
+                    "size": 32,
+                    "style": ""
+                },
+                "outline": false,
+                "text": "Kubernetes Community Meeting starting in a few minutes..."
+            },
+            "sync": 0,
+            "volume": 1.0
+        },
+        {
+            "balance": 0.5,
+            "deinterlace_field_order": 0,
+            "deinterlace_mode": 0,
+            "enabled": true,
+            "flags": 0,
+            "hotkeys": {},
+            "id": "image_source",
+            "mixers": 0,
+            "monitoring_type": 0,
+            "muted": false,
+            "name": "comm splash",
+            "private_settings": {},
+            "push-to-mute": false,
+            "push-to-mute-delay": 0,
+            "push-to-talk": false,
+            "push-to-talk-delay": 0,
+            "settings": {
+                "file": "/home/jorge/Downloads/nick-karvounis-75581-unsplash.jpg"
```

Comment:
All the json probably needs to be scrubbed for personal system references. 
Do you know if OBS will do relative paths? If so could set everything relative to the dir.

Reply:
Yeah I'm wondering if we can stick the splash screen assets in here and have OBS be able to use those instead of my local paths?

Reply:
I'm pretty sure they would be. Also +1 for getting the shareable assets together in one place.